docs/Alphaville.txt

```diff
@@ -0,0 +1,6 @@
+
+Dilca
```

Comment:
is this file strictly needed ?

Reply:
My notes on last conversation with Gerardo, we can drop it.